Rakefile

```diff
@@ -1,3 +1,6 @@
+require "rubygems"
+require "bundler/setup"
+
```

Comment:
Lets keep '/" style consistent here. Either change 'em all or follow what is in there.